pkg/server/debug.go

```diff
@@ -83,6 +88,24 @@ func authRequest(r *http.Request) (allow, sensitive bool) {
 	return allow, sensitive
 }
 
+func handleVModule(w http.ResponseWriter, r *http.Request) {
+	w.Header().Set("Content-Type", "text/plain; charset=utf-8")
+	if !ClusterSettings.DebugVModule.Get() {
+		http.Error(
+			w,
+			"not allowed (due to the 'server.remote_debugging.vmodule' setting)",
+			http.StatusForbidden,
+		)
+		return
+	}
+	spec := r.RequestURI[len(vmodulePrefix):]
+	if err := log.SetVModule(spec); err != nil {
+		http.Error(w, err.Error(), http.StatusInternalServerError)
```

Comment:
should there be a `return` after this line? looks like the original code was missing this.